test/org/sagebionetworks/bridge/services/ScheduledActivityServiceRecurringTest.java

```diff
@@ -91,6 +91,9 @@ public void after() {
         }
     }
     
+    // WARNING: The notes here are not correct due to changes we've made in the way we test the time zone.
+    // The last day MSK seems to fall outside the scope of the tests. This needs to be fixed in later
+    // updates but we're currently addressing the duplication of tasks.
     @Test
     public void retrievalActivitiesAcrossTimeAndTimeZones() throws Exception {
```

Comment:
Moved enrollment so it is at a fixed time (the time frozen in the test -2 days). Added back some tasks that will appear when event is at that time.